.github/workflows/test.yaml

```diff
@@ -0,0 +1,32 @@
+name: test
+on: pull_request
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          override: true
+          toolchain: stable
+          profile: minimal
+      - uses: Swatinem/rust-cache@v2
+      - name: Install jq
+        uses: dcarbone/install-jq-action@v1.0.1
+      - name: 'Check jq'
+        run: |
+          which jq
+          jq --version
+          export JQ_LIB_DIR=$(eval which jq)
```

Comment:
Currently, jq-rs targets jq 1.6 and if it happens to work with 1.7 it'll be by accident. 
Can we pin the jq version at 1.6 with this action? I'd like it to be explicit for now.

Reply:
currently the jq action exists only for 1.6 i think https://github.com/marketplace/actions/install-jq
i have pinned the version to 1.6 now in my recent commit.